specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/StaticSites.json

```diff
@@ -2688,6 +2688,127 @@
     }
   },
   "definitions": {
+    "ArmPlan": {
```

Comment:
>ArmPlan [](start = 5, length = 7)

this is a dup of /common-types/resource-management/v2/types.json#Plan right? please ref it instead